_posts/2010-05-07-odata-roundup-3.html

```diff
@@ -14,10 +14,10 @@
 <div>
 <p>Many exciting things have happened in the OData world since our last post.</p>
 <ul>
-<li>We <a href="https://www.odata.org/blog/the-open-data-protocol-net-framework-client-library-source-code-available-for-download"> released the source code</a> for our .NET client library on <a href="http://odata.codeplex.com/">codeplex</a>.</li>
-<li>We announced the <a href="https://www.odata.org/blog/odata-roadshow">OData Roadshow</a> which will visit: New York, Chicago, Mountain View, Shanghai, Tokyo, Reading and Paris in the coming months. See if you can get along.</li>
+<li>We <a href="/blog/the-open-data-protocol-net-framework-client-library-source-code-available-for-download"> released the source code</a> for our .NET client library on <a href="http://odata.codeplex.com/">codeplex</a>.</li>
+<li>We announced the <a href="/blog/odata-roadshow">OData Roadshow</a> which will visit: New York, Chicago, Mountain View, Shanghai, Tokyo, Reading and Paris in the coming months. See if you can get along.</li>
 <li>The <a href="http://windowsteamblog.com/blogs/windowslive/archive/2010/04/29/messenger-across-the-web.aspx"> next version of Windows Live Messenger</a> will include a RESTful interface that supports OData.</li>
```

Comment:
Link http://windowsteamblog.com/blogs/windowslive/archive/2010/04/29/messenger-across-the-web.aspx
redirects to https://blogs.windows.com/